examples/eth1_with_sriov.yml

```diff
@@ -5,6 +5,7 @@ interfaces:
     state: up
     ethernet:
       sr-iov:
+        drivers-autoprobe: false
```

Comment:
I guess most of SRIOV user are expecting a VF been created. Hence I would suggest remove this change to example YAML.